parse_lib.py

```diff
@@ -95,8 +94,9 @@ def resolve_relative_resource_urls(html_string):
     imgs = html.find_all("img", src=True)
     equations = html.find_all("object", data=True)
     list(map(update_anchor_href, anchors))
+    equations_as_imgs = list(map(partial(convert_to_img, html), equations))
```

Comment:
Sure thing, added in.

On that note... at the risk of having some stale comments down the line, I think it might be nice to add docstrings in some of these functions (particularly `parse_lib`, since the functions here aren't really localized and it's hard to just read it and figure out what's happening). How do you feel about this?